sift.hpp

```diff
@@ -7,8 +7,8 @@
 #include "types.hpp"
 #include "matrix.hpp"
 
-using img_epochs = std::vector<std::vector<vigra::MultiArray<2, f32_t>>>;
-using interest_point_epochs = std::vector<std::vector<Matrix<f32_t>>>; 
+using img_epochs = Matrix<vigra::MultiArray<2, f32_t>>;
+using interest_point_epochs = Matrix<Matrix<f32_t>>; 
 
```

Comment:
Finde ich überhaupt nicht, da sie gar nichts (für mich) aussagen zu dem was dahinter steckt. Bei `img_epochs` geb ich dir trotzdem Recht, das Ding ist zu crass lang, aber das andere, speziell weil es meine Matrix geschachtelt in meiner Matrix ist :heart: , würde ich gerne direkt ohne alias benutzen.